entrypoint.sh

```diff
@@ -288,11 +314,14 @@ function seiso_lint() {
 
 	while read -r file; do
 		if [[ -n ${FILTER_REGEX_EXCLUDE:+x} && "${file}" =~ ${FILTER_REGEX_EXCLUDE} ]]; then
+			feedback DEBUG "${file} matched the exclusion regex of ${FILTER_REGEX_EXCLUDE}"
 			excluded+=("${file}")
 			continue
+		else
+			feedback DEBUG "${file} didn't match the exclusion regex of ${FILTER_REGEX_EXCLUDE:-not set}"
 		fi
 		included+=("${file}")
-	done < <(find . \( -path "./.git" -prune \) -o \( -type f -print \))
+	done < <(find /goat \( -path "./.git" -prune \) -o \( -type f -print \))
```

Comment:
This is needed so we can do the more explicit `^/{{.PROJECT_SLUG}}/goat/.*` exclusion